packages/connect-thegraph-tokens/subgraph/schema.graphql

```diff
@@ -2,7 +2,7 @@ type TokenManager @entity {
   id: ID!
   address: Bytes!
   orgAddress: Bytes!
-  token: MiniMeToken! @derivedFrom(field: "tokenManager")
+  token: MiniMeToken @derivedFrom(field: "tokenManager")
```

Comment:
A TokenManager's token can now be null, since it accounts for cases of a TokenManager app that has not been initialized.